specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -236,40 +236,72 @@ model ShortAnswerOptions {
 
 @doc("Human-readable error code.")
 enum InnerErrorCode {
+    @doc("Human-readable error code.")
```

Comment:
I see these didn't have descriptions before. In general, I'd say feel free to suppress these warnings instead, and link to a tracking bug number for the service team to add them eventually. Thus, the warnings to add useful docs aren't unduly suppressed in a way they'd go unnoticed.